tests/unit/conditions/test_return_value.py

```diff
@@ -23,6 +23,47 @@
 from nucypher.policy.conditions.lingo import ReturnValueTest
 
 
+def test_return_value_key():
+    test = ReturnValueTest(comparator=">", value="0", key="james")
+    assert test.eval({"james": 1})
+    assert not test.eval({"james": -1})
+
+    with pytest.raises(KeyError):
+        test.eval({"bond": 1})
+
+    test = ReturnValueTest(comparator=">", value="0", key=4)
+    assert test.eval({4: 1})
+    assert not test.eval({4: -1})
+
+    with pytest.raises(KeyError):
+        test.eval({5: 1})
+
+
+def test_return_value_index():
+    test = ReturnValueTest(comparator=">", value="0", key=0)
+    assert test.eval([1])
+    assert not test.eval([-1])
+
+    test = ReturnValueTest(comparator="==", value='"james"', key=3)
+    assert test.eval([0, 1, 2, '"james"'])
+
+    with pytest.raises(IndexError):
+        test.eval([0, 1, 2])
+
+
+def test_return_value_index_tuple():
+    test = ReturnValueTest(comparator=">", value="0", key=0)
+    assert test.eval((1,))
+    assert not test.eval((-1,))
```

Comment:
This implies that `0` is equal to `"0"` - is this intentional, or just a side-effect of python's `eval`?  I'm aware that this behavior predates this PR, but it's illuminated here.

Reply:
I believe this is to do with `eval` and is related to why `'"james"'` is needed in the above comment

Reply:
Is an exception raised when using single quotes (perhaps a `NameError` or `TypeError`)?